components/eamxx/tests/coupled/dynamics_physics/homme_mam4xx_pg2/CMakeLists.txt

```diff
@@ -85,3 +85,32 @@ CompareNCFilesFamilyMpi (
   LABELS dynamics physics mam4xx
   META_FIXTURES_REQUIRED ${FIXTURES_BASE_NAME}_npMPIRANKS_omp1
 )
+
+if (SCREAM_ENABLE_BASELINE_TESTS)
```

Comment:
I thought about making a cmake function to create the baseline test. I may add it to this PR maybe.